src/vmm/src/builder.rs

```diff
@@ -1547,13 +1490,6 @@ pub mod tests {
         tmp_sock_file.remove().unwrap();
         let vsock_config = default_config(&tmp_sock_file);
 
-        let mut cmdline = default_kernel_cmdline();
-        insert_vsock_device(&mut vmm, &mut cmdline, &mut event_manager, vsock_config);
-        // Check if the vsock device is described in kernel_cmdline.
-        #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-        assert!(cmdline_contains(
-            &cmdline,
-            "virtio_mmio.device=4K@0xd0000000:5"
-        ));
+        insert_vsock_device(&mut vmm, &mut event_manager, vsock_config);
```

Comment:
This test does not actually assert anything anymore now, so I think we can remove it altogether